packages/mikado_reborn/src/components/Tooltip/Tooltip.scss

```diff
@@ -15,6 +15,7 @@
     border-radius: 12px;
     background: map-get($colors, 'neutral');
     color: map-get($colors, 'white');
+    z-index: 300;
```

Comment:
On a décidé d'une échelle pour la profondeur du site?

Reply:
I assumed we want tooltip over snackbar but we don't have any specs for that yet :/

Reply:
All right!! let's take some times to define this sacle ;-)